<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HelpViewer</class>
 <widget class="QMainWindow" name="HelpViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Help â€“ Coquery</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTextBrowser" name="index">
       <property name="openLinks">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QTextBrowser" name="content">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_prev"/>
   <addaction name="action_next"/>
   <addaction name="action_home"/>
   <addaction name="separator"/>
   <addaction name="action_print"/>
   <addaction name="separator"/>
   <addaction name="action_zoom_in"/>
   <addaction name="action_zoom_out"/>
   <addaction name="action_reset_zoom"/>
  </widget>
  <action name="action_next">
   <property name="icon">
    <iconset theme="go-next">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Forward</string>
   </property>
  </action>
  <action name="action_prev">
   <property name="icon">
    <iconset theme="go-previous">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Previous</string>
   </property>
   <property name="toolTip">
    <string>Previous</string>
   </property>
  </action>
  <action name="action_home">
   <property name="icon">
    <iconset theme="go-home">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </action>
  <action name="action_zoom_in">
   <property name="icon">
    <iconset theme="zoom-in">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Zoom in</string>
   </property>
   <property name="toolTip">
    <string>Zoom in</string>
   </property>
  </action>
  <action name="action_zoom_out">
   <property name="icon">
    <iconset theme="zoom-out">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Zoom out</string>
   </property>
  </action>
  <action name="action_reset_zoom">
   <property name="icon">
    <iconset theme="zoom-original">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Reset zoom</string>
   </property>
  </action>
  <action name="action_print">
   <property name="icon">
    <iconset theme="document-print">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
